packages/wpcom-template-parts/package.json

```diff
@@ -0,0 +1,47 @@
+{
+	"name": "@automattic/wpcom-template-parts",
+	"version": "1.0.0",
+	"description": "WordPress.com template parts to be used everywhere",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"calypso:src": "src/index.ts",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/wpcom-template-parts"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"private": true,
+	"bugs": "https://github.com/Automattic/wp-calypso/issues",
+	"types": "dist/types",
+	"scripts": {
+		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
+		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json",
+		"prepack": "yarn run clean && yarn run build",
+		"watch": "tsc --build ./tsconfig.json --watch"
+	},
+	"dependencies": {
+		"@automattic/components": "workspace:^",
```

Comment:
Let's make sure we're including all dependencies. For example, I see we're using `i18n -calypso` and it's not specifically listed here.